single-line-diagram/single-line-diagram-core/src/main/resources/ConvergenceLibrary/unknown-component.svg

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg xmlns="http://www.w3.org/2000/svg" version="1.0" height="16" width="16">
```

Comment:
Fix width and height to 20, even if they are ignored when generating the diagram it's used when previewing the component